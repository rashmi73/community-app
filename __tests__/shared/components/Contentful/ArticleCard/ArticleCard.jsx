import React from 'react';
import Renderer from 'react-test-renderer/shallow';
import ArticleCardInner from 'components/Contentful/ArticleCard/ArticleCard';

test('Matches shallow shapshot', () => {
  const renderer = new Renderer();
  const MOCK_PROPS = {
    id: '1',
    article: {
      title: 'Lorem ipsum',
      content: 'Some words',
      tags: ['TCO'],
      readTime: '5 min read',
      creationDate: '2018-12-25',
      upvotes: undefined,
      commentsCount: undefined,
    },
    articleCard: {},
    contentAuthor: null,
    featuredImage: null,
    theme: {},
    themeName: 'Article large',
  };
  renderer.render(<ArticleCardInner {...MOCK_PROPS} />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
