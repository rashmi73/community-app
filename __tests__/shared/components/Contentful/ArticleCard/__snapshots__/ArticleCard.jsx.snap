// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches shallow shapshot 1`] = `
<ArticleCard
  article={
    Object {
      "commentsCount": undefined,
      "content": "Some words",
      "creationDate": "2018-12-25",
      "readTime": "5 min read",
      "tags": Array [
        "TCO",
      ],
      "title": "Lorem ipsum",
      "upvotes": undefined,
    }
  }
  articleCard={Object {}}
  contentAuthor={null}
  featuredImage={null}
  id="1"
  theme={Object {}}
  themeName="Article large"
/>
`;
